app/components/toasts/undo.js

```diff
@@ -0,0 +1,64 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { task, timeout } from 'ember-concurrency';
+import { htmlSafe } from '@ember/template';
+
+export default class ToastsUndoComponent extends Component {
+  @service() currentRecognition;
+  @service() router;
+  @service() toaster;
+  @tracked isLoading = false;
+  @tracked progress = 0;
+
+  constructor() {
+    super(...arguments);
+    this.animateProgress.perform();
+  }
+  get progressStyle() {
+    const width = `${this.progress}%`;
+    const height = '2px';
+    const backgroundColor = 'var(--au-green-500)';
+    const position = 'absolute';
+    const bottom = '0';
+    const left = '0';
+
+    const styleString = `width: ${width}; height: ${height}; background-color: ${backgroundColor}; position: ${position}; bottom: ${bottom}; left: ${left};`;
+
+    return htmlSafe(styleString);
+  }
+  @task
+  *animateProgress() {
```

Comment:
If you are in the same ember version as us, that I think you are this is no longer how you create tasks